exercises/acronym/acronym_test.go

```diff
@@ -0,0 +1,47 @@
+package acronym
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input     string
+	expected  string
+	expectErr bool
+}
+
+var stringTestCases = []testCase{
+	{"Portable Network Graphics", "PNG", false},
+	{"HyperText Markup Language", "HTML", false},
+	{"Ruby on Rails", "ROR", false},
+	{"PHP: Hypertext Preprocessor", "PHP", false},
+	{"First In, First Out", "FIFO", false},
+	{"Complementary metal-oxide semiconductor", "CMOS", false},
+}
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
+}
+
+func TestAcronym(t *testing.T) {
+	for _, test := range stringTestCases {
+		actual, actualErr := abbreviate(test.input)
+		if !test.expectErr {
+			if actualErr != nil {
+				// if we don't expected error but got error
+				var _ error = actualErr
```

Comment:
Seems good to me. In fd8572684c4138fda1b2d583bdaea09dec2051cb it may have been deceptive since it was under `case err != nil`, but that's because I couldn't have placed it before the `case`